python/tests/test_build_disk_index.py

```diff
@@ -3,6 +3,7 @@
 
 import time
 import argparse
+import diskannpy
```

Comment:
what is your intent of importing the module namespace?  I'm not against it, it just seems unnecessary given the `from diskannpy import ...` down below.

Reply:
Line 22 is ```index = DiskANNFloatIndex(diskannpy.L2)``` which uses the module import. Alternative would be do a ```from diskannpy import L2```

Reply:
ah ok!  thanks for clarifying, I appreciate it!